src/scrabble.js

```diff
@@ -1,5 +1,76 @@
-function scrabble() {
+const createLetterValue = () => {
```

Comment:
I like the idea of dynamically creating the object but I am unsure how easy it would be to maintain in the case changes need to be made. Personally I prefer having an object in a separate file that holds all the keys and values in case an adjustments would need to be made.

Reply:
This is what I do in a way. I'm just creating the object in a function